plugins/outputs/cloudwatchlogs/cloudwatchlogs.go

```diff
@@ -127,6 +128,12 @@ func (c *CloudWatchLogs) getDest(t Target) *cwDest {
 		},
 	)
 	client.Handlers.Build.PushBackNamed(handlers.NewRequestCompressionHandler([]string{"PutLogEvents"}))
+
+	containerInsightRegexp := "^/aws/.*containerinsights/.*/(performance|prometheus)$"
```

Comment:
This looks like a constant. I know that we call `getDest` more than once (I believe once per log file?). We should move the regular expression outside of this function so that we don't have to compile it every time. Not sure exactly what the syntax would look like, but I think it'd be best as a top-level `var` or `const` for the Regexp struct like:
```go
const (
    containerInsightExpression := "^/aws/.*containerinsights/.*/(performance|prometheus)$"
)

var (
    containerInsightExpressionP := regexp.MustCompile(containerInsightExpression)
)
```

Or something like that. It might not work exactly like that, but the point is I think we can refactor this so that we don't have to compile it every time the `getDest` function is called.

Reply:
Good call! I've moved the regex to the top level and reuse it. Thanks!